modules/iam-profile/main.tf

```diff
@@ -72,7 +72,20 @@ resource "aws_iam_policy" "ReadOnlyPolicy" {
               "kinesis:DescribeStream",
               "servicecatalog:SearchProducts",
               "servicecatalog:DescribeProduct",
-              "servicecatalog:DescribePortfolio"
+              "servicecatalog:DescribePortfolio",
+              "eks:ListNodegroups",
+              "eks:DescribeFargateProfile",
+              "eks:ListTagsForResource",
+              "eks:ListAddons",
+              "eks:DescribeAddon",
+              "eks:ListFargateProfiles",
+              "eks:DescribeNodegroup",
+              "eks:DescribeIdentityProviderConfig",
+              "eks:ListUpdates",
+              "eks:DescribeUpdate",
+              "eks:ListIdentityProviderConfigs",
+              "servicecatalog:DescribeServiceAction",
+              "servicecatalog:DescribeProvisioningArtifact"
```

Comment:
Where are these coming from? And we should keep these alphabetically ordered

Reply:
@spasam 
based on this https://github.com/Uptycs/uptycs/commit/3dee14328f6df1145960de1d26099763ac5f13f2 i added these permissions